corehq/apps/hqadmin/reports.py

```diff
@@ -1166,3 +1168,70 @@ def _create_row(self, log, *args, **kwargs):
         row = super(DeviceLogSoftAssertReport, self)._create_row(log, *args, **kwargs)
         row.append(log.domain)
         return row
+
+
+class CommCareVersionReport(AdminReport, ElasticTabularReport):
+    slug = "commcare_version"
+    name = ugettext_lazy("CommCare Version")
+
+    @property
+    def headers(self):
+        versions = get_all_versions()
+        headers = DataTablesHeader(
+            DataTablesColumn(_("Project"))
+        )
+        for version in versions:
+            headers.add_column(DataTablesColumn(version))
+        return headers
+
+    @property
+    def rows(self):
+        versions = get_all_versions()
+
+        def get_query(version):
+            return {
+                "query": {
+                    "bool": {
+                        "must": [
+                            {
+                                "range": {
+                                    "received_on": {
+                                        "gte": "now-90d",
+                                        "lte": "now"
+                                    }
+                                }
+                            },
+                            {
+                                "query_string": {
+                                    "default_field": "appVersion",
+                                    "query": "*" + version + "*"
+                                }
+                            }
+                        ]
+                    }
+                },
+                "aggs": {
+                    "by_domain": {
+                        "terms": {
+                            "field": "domain"
+                        }
+                    }
+                }
+            }
+
+        rows = {}
+        for domain in es_domain_query(show_stats=False)['hits']['hits']:
```

Comment:
Will this loop through every domain on HQ? https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqadmin/reports.py#L671 passes in facets and size for proper pagination